recipe/meta.yaml

```diff
@@ -1,20 +1,23 @@
+{% set name = "bdw-gc" %}
 {% set version = "7.6.4" %}
+{% set sha256 = "b9f30bd5759e4282fdb161414a1b23845c63b725e8ed40fc110d0859a4396b81"%}
 
 package:
-  name: bdw-gc
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: gc7_4_2.tar.gz
-  url: https://github.com/ivmai/bdwgc/archive/gc7_4_2.tar.gz
-  sha1: 441777b9865a730c9c804f5f5de2aa9a55fc9419
+  fn: gc-{{ version }}.tar.gz
+  url: https://github.com/ivmai/bdwgc/releases/download/v{{ version }}/gc-{{ version }}.tar.gz
```

Comment:
`https://github.com/ivmai/bdwgc/archive/v{{ version }}.tar.gz` ?

Reply:
Yes, saw it a second ago since the sha256 didn't work...thanks :)